blt/src/Commands/HumsciTrait.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Acquia\Blt\Custom\Commands;
+
+/**
+ * Trait HumsciTrait.
+ *
+ * @package Acquia\Blt\Custom\Commands
+ */
+trait HumsciTrait {
+
+  /**
+   * Recusive glob.
+   *
+   * @param string $pattern
+   *   Glob pattern.
+   * @param int $flags
+   *   Globl flags.
+   *
+   * @return array|void
+   *   Response from glob.
+   */
+  protected function rglob($pattern, $flags = 0) {
+    $files = glob($pattern, $flags);
```

Comment:
This should be replaced with the API version: https://api.drupal.org/api/drupal/vendor%21zendframework%21zend-stdlib%21src%21Glob.php/function/Glob%3A%3Aglob/8.5.x